src/components/Filter-time/style.css

```diff
@@ -0,0 +1,46 @@
+.filter{
```

Comment:
remove the css here, and add tailwind classes to the code instead, all of this is feasible by using only tailwind